[🐞] `npm run qwik add express` does not create npm `serve` script

### Which component is affected?

Starters / CLI

### Describe the bug

Running `npm run qwik add express` to add the node integration [as described in the docs](https://qwik.builder.io/qwikcity/adaptors/node/) adds this section below to the README, however the `serve` script is not added to `package.json` and so the provided instructions do not work.

```
## Express Server
 
This app has a minimal [Express server](https://expressjs.com/) implementation. After running a full build, you can preview the build using the command:
 
npm run serve

Then visit [http://localhost:8080/](http://localhost:8080/)
```

### Reproduction

Reproduction steps provided

### Steps to reproduce

```sh
npm init qwik@latest // accept any defaualts, build into "qwik" directory for the next step to work
cd wqik
npm run qwik add express // confirm apply changes
npm run serve
```

The following missing script output is created.

```
npm ERR! Missing script: "serve"
npm ERR! 
npm ERR! To see a list of scripts, run:
npm ERR!   npm run

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/liam/.npm/_logs/2022-12-11T21_03_53_425Z-debug-0.log
```

### System Info

```shell
System:
    OS: Linux 5.15 Ubuntu 22.04.1 LTS 22.04.1 LTS (Jammy Jellyfish)
    CPU: (4) x64 Intel(R) Core(TM) i7-5500U CPU @ 2.40GHz
    Memory: 1.95 GB / 7.67 GB
    Container: Yes
    Shell: 5.1.16 - /bin/bash
  Binaries:
    Node: 18.12.1 - ~/.nvm/versions/node/v18.12.1/bin/node
    Yarn: 1.6.0 - /usr/bin/yarn
    npm: 8.19.2 - ~/.nvm/versions/node/v18.12.1/bin/npm
  Browsers:
    Chrome: 102.0.5005.61
    Firefox: 107.0
  npmPackages:
    @builder.io/qwik: 0.15.2 => 0.15.2 
    @builder.io/qwik-city: 0.0.128 => 0.0.128 
    vite: 3.2.4 => 3.2.4
```


### Additional Information

_No response_

--- a/packages/docs/src/routes/qwikcity/adaptors/azure-swa/index.mdx
+++ b/packages/docs/src/routes/qwikcity/adaptors/azure-swa/index.mdx
@@ -26,7 +26,7 @@ The adaptor will add a new `vite.config.ts` within the `adaptors/` directory, an
     └── entry.azure-swa.tsx
 ```
 
-Additionally, within the `package.json`, the `build.server` script will be updated with the Azure Static Web Apps build.
+Additionally, within the `package.json`, the `build.server` and `deploy` scripts will be updated.
 
 ## Production build
 
@@ -36,18 +36,9 @@ To build the application for production, use the `build` command, this command w
 npm run build
 ```
 
-## Local development deployment
-
-To deploy the application for local development:
-
-```shell
-npm run build
-npm run serve
-```
-
 ## Deploy to Azure
 
-After installing the integration using `npm run qwik add` the project is ready to be deployed to Azure Static Web Apps.
+After installing the integration using `npm run qwik add azure-swa` the project is ready to be deployed to Azure Static Web Apps.
 
 There are two ways to deploy:
 

--- a/packages/docs/src/routes/qwikcity/adaptors/cloudflare-pages/index.mdx
+++ b/packages/docs/src/routes/qwikcity/adaptors/cloudflare-pages/index.mdx
@@ -26,7 +26,7 @@ The adaptor will add a new `vite.config.ts` within the `adaptors/` directory, an
     └── entry.cloudflare-pages.tsx
 ```
 
-Additionally, within the `package.json`, the `build.server` script will be updated with the Cloudflare build.
+Additionally, within the `package.json`, the `build.server` and `deploy` scripts will be updated.
 
 ## Production build
 
@@ -38,18 +38,9 @@ npm run build
 
 [Read the full guide here](https://github.com/BuilderIO/qwik/tree/main/starters/adaptors/cloudflare-pages/README.md)
 
-## Local development deployment
-
-To deploy the application for local development:
-
-```shell
-npm run build
-npm run serve
-```
-
 ## Deploy to Cloudflare Pages
 
-After installing the integration using `npm run qwik add`. The project is ready to be deployed to Cloudflare Pages. However, you will need to create a git repository and push the code to it.
+After installing the integration using `npm run qwik add cloudflare-pages`, the project is ready to be deployed to Cloudflare Pages. However, you will need to create a git repository and push the code to it.
 
 Please refer to the [Cloudflare Pages docs](https://developers.cloudflare.com/pages/framework-guides/deploy-a-qwik-site/) for more information on how to deploy your site.
 

--- a/packages/docs/src/routes/qwikcity/adaptors/netlify-edge/index.mdx
+++ b/packages/docs/src/routes/qwikcity/adaptors/netlify-edge/index.mdx
@@ -28,7 +28,7 @@ The adaptor will add a new `vite.config.ts` within the `adaptors/` directory, an
     └── entry.netlify-edge.tsx
 ```
 
-Additionally, within the `package.json`, the `build.server` script will be updated with the Netlify Edge build.
+Additionally, within the `package.json`, the `build.server` and `deploy` scripts will be updated.
 
 ## Production build
 
@@ -52,7 +52,7 @@ Notice that you might need a [Netlify account](https://docs.netlify.com/get-star
 
 ## Production deploy
 
-After installing the integration using `npm run qwik add`. The project is ready to be deployed to Netlify. However, you will need to create a git repository and push the code to it.
+After installing the integration using `npm run qwik add netlify-edge`, the project is ready to be deployed to Netlify. However, you will need to create a git repository and push the code to it.
 
 Please refer to the Netlify docs for more information on how to deploy your site: [Netlify docs](https://docs.netlify.com/site-deploys/create-deploys/)
 

--- a/packages/docs/src/routes/qwikcity/adaptors/vercel-edge/index.mdx
+++ b/packages/docs/src/routes/qwikcity/adaptors/vercel-edge/index.mdx
@@ -28,7 +28,7 @@ The adaptor will add a new `vite.config.ts` within the `adaptors/` directory, an
     └── entry.vercel-edge.tsx
 ```
 
-Additionally, within the `package.json`, the `build.server` script will be updated with the Vercel Edge build.
+Additionally, within the `package.json`, the `build.server` and `deploy` scripts will be updated.
 
 ## Production build
 
@@ -52,7 +52,7 @@ Notice that you might need a [Vercel account](https://vercel.com/docs/concepts/g
 
 ## Production deploy
 
-After installing the integration using `npm run qwik add`. The project is ready to be deployed to Vercel. However, you will need to create a git repository and push the code to it.
+After installing the integration using `npm run qwik add vercel-edge`, the project is ready to be deployed to Vercel. However, you will need to create a git repository and push the code to it.
 
 Please refer to the Vercel docs for more information on how to deploy your site: [Vercel docs](https://vercel.com/docs/concepts/deployments/overview)
 

